ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -30,7 +30,44 @@ esac
 
 dir=/tmp/installer
 mkdir "${dir}/"
-cp "${SHARED_DIR}/install-config.yaml" "${dir}/"
+
+function set_install_config_one_replica () {
```

Comment:
Do we have ``yq`` in this image?
Then it might become prettier

Reply:
Or maybe install it as you do with ``pyyaml``

Reply:
I considered that, I think Python is a better fit here than `yq` - there's a lot of logic here to support various edge-cases caused by the variety of platforms that generate the `install-config.yaml` before this step - sometimes they don't define controlPlane or don't define compute - I'm not sure how simple that is to handle with `yq`

Reply:
yq can either add or update fields, depending on if they already exist on the document
Give it a try, it will make the code more explicit and without the python implementation detail. Also it will allow extensions in the future without changing the Python code

Reply:
Doesn't seem like there's a simple way to install `yq` here - the version in `pip` is rather old and with a different syntax (even less similar to the familiar `jq`), I don't think relying on that version is a good idea - they've already deprecated V3 (aug 21) - the version in pip is V2. I'll keep it Python for now unless you can come up with a way to install it

Reply:
https://github.com/mikefarah/yq#plain-binary

Reply:
As noted - in your own specific configuration step, you are free to use any container image you need, which can include `yq`.

Reply:
I've decided to avoid yq altogether due to:

> Or maybe it'll be a good idea to avoid it altogether, seeing that it's rather unstable and they keep deprecating the syntax

And in this case, I'm modifying an existing installer configuration step - it's not mine, I prefer to avoid modifications to its Dockerfile.